src/containers/CreatePipelineRun/CreatePipelineRun.js

```diff
@@ -57,7 +58,8 @@ const initialState = {
   submitError: '',
   timeout: '60',
   validationError: false,
-  validTimeout: true
+  validTimeout: true,
+  pendingPipelineStatus: ''
```

Comment:
nit: put this above `pipelineError` so they're in alphabetical order, makes it easier to find things in the list and avoid duplicates etc.